firstvoices/backend/serializers/job_serializers.py

```diff
@@ -36,28 +31,23 @@ class Meta:
         )
 
 
-class CustomOrderRecalculationResultSerializer(serializers.ModelSerializer):
-    site = serializers.StringRelatedField()
-    current_task_status = serializers.SerializerMethodField()
-    latest_recalculation_result = serializers.SerializerMethodField()
-
-    @staticmethod
-    @extend_schema_field(OpenApiTypes.STR)
-    def get_current_task_status(obj):
-        async_result = AsyncResult(obj.task_id)
-        return async_result.status
+class DictionaryCleanupJobSerializer(
```

Comment:
Suggestion: could we use the same serializer for the real cleanup and the preview if we called the result a "result"?

Reply:
As per our discussion separate serializers are still required but to the difference in URL for the separate views. However the `DictionaryCleanupJobSerializer` has been updated such that the `DictionaryCleanupPreviewJobSerializer` only needs one line of code.